Recipes/Along-slope-velocities.ipynb

```diff
@@ -23,6 +23,7 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    salt_ds = esm_datastore.search(path='/g/data/ik11/outputs/access-om2-01/01deg_jra55v13_ryf9091/*',variable='salt',frequency='1mon').to_dataset_dict()
</pre><p>Again, avoid searching on filenames (or other 'magic things' we know because we're experiment experts)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/458/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>